v2/subaccount_service.go

```diff
@@ -0,0 +1,76 @@
+package binance
+
+import (
+	"context"
+	"encoding/json"
+)
+
+// TransferToSubAccountService transfer to subaccount
+type TransferToSubAccountService struct {
+	c       *Client
+	toEmail string
+	asset   string
+	amount  string
+}
+
+// ToEmail set toEmail
+func (s *TransferToSubAccountService) ToEmail(toEmail string) *TransferToSubAccountService {
+	s.toEmail = toEmail
+	return s
+}
+
+// Asset set asset
+func (s *TransferToSubAccountService) Asset(asset string) *TransferToSubAccountService {
+	s.asset = asset
+	return s
+}
+
+// Amount set amount
+func (s *TransferToSubAccountService) Amount(amount string) *TransferToSubAccountService {
+	s.amount = amount
+	return s
+}
+
+func (s *TransferToSubAccountService) transferToSubaccount(ctx context.Context, endpoint string, opts ...RequestOption) (data []byte, err error) {
+	r := &request{
+		method:   "POST",
+		endpoint: endpoint,
+		secType:  secTypeSigned,
+	}
+	m := params{
+		"toEmail": s.toEmail,
+		"asset":   s.asset,
+		"amount":  s.amount,
+	}
+	r.setParams(m)
+	data, err = s.c.callAPI(ctx, r, opts...)
+	if err != nil {
+		return []byte{}, err
+	}
+	return data, nil
+}
+
+// Do send request
+func (s *TransferToSubAccountService) Do(ctx context.Context, opts ...RequestOption) (res *TransferToSubAccountResponse, err error) {
+	data, err := s.transferToSubaccount(ctx, "/sapi/v1/sub-account/transfer/subToSub", opts...)
+	if err != nil {
+		return nil, err
+	}
+	res = &TransferToSubAccountResponse{}
+	err = json.Unmarshal(data, res)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
+}
+
+// Test send test api to check if the request is valid
+func (s *TransferToSubAccountService) Test(ctx context.Context, opts ...RequestOption) (err error) {
```

Comment:
Hi @nandubatchu , I can't find the endpoint `/test` from [binance API](https://binance-docs.github.io/apidocs/spot/en/#margin-transfer-for-sub-account-for-master-account), is this an official API?
If you want to test the API, it's better to add unit test for it.

Reply:
Let me remove this /test method.